app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Since in the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram, the user is represented in `Comment, Like and Post` tables as `author`, represented by the AuthorId, it will be nice to reflect that in the implementation of your model classes. Some like so: 

```
belongs_to :author, class_name: 'User'
belongs_to :post
```
Please kindly fix that.